erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -704,10 +704,34 @@ def get_items_for_material_requests(doc, ignore_existing_ordered_qty=None):
 @frappe.whitelist()
 def get_item_data(item_code):
 	item_details = get_item_details(item_code)
+	bom_no = item_details.get("bom_no")
+	
+	total_operating_time = frappe.db.sql("""
+		SELECT
+			sum(bo.time_in_mins) AS total_operating_time
+		FROM
+			`tabBOM Operation` AS bo 
+		WHERE
+			bo.parent = %s
+	""", (bom_no))
+	total_operating_time = flt(total_operating_time[0][0]) / 60.0
```

Comment:
Avoid using direct SQL queries wherever you can, as they are prone to injection attacks, and especially since this can be accessed from the client-side directly

```suggestion
	total_operating_hours = frappe.db.get_value("BOM Operation", {"parent": bom_no}, "sum(bo.time_in_mins) AS total_operating_time")
	total_operating_hours = flt(total_operating_time) / 60.0
```